src/common/Configuration/Config.cpp

```diff
@@ -57,6 +58,81 @@ namespace
 
         return true;
     }
+
+    // Converts ini keys to the environment variable key (upper snake case).
+    // Example of conversions:
+    //   SomeConfig => SOME_CONFIG
+    //   myNestedConfig.opt1 => MY_NESTED_CONFIG_OPT_1
+    //   LogDB.Opt.ClearTime => LOG_DB_OPT_CLEAR_TIME
+    std::string IniKeyToEnvVarKey(std::string const& key)
+    {
+        std::string result = "";
```

Comment:
```suggestion
        std::string result;
```
Clang-Tidy: Redundant string initialization